ui/src/adapters/api/issuer-state.ts

```diff
@@ -111,28 +104,15 @@ export async function getTransactions({
       signal,
       url: `${API_VERSION}/state/transactions`,
     });
-    const { data } = resultOKTransactionListParser.parse(response);
-
-    return {
-      data: {
-        failed: data.failed,
-        successful: data.successful.sort(
-          ({ publishDate: a }, { publishDate: b }) => dayjs(b).unix() - dayjs(a).unix()
-        ),
-      },
-      isSuccessful: true,
-    };
+    return buildSuccessResponse(
+      getListParser(transactionParser)
```

Comment:
Do you think `getListParser` could do the sorting by accepting the key?

Reply:
I thought about that and discarded the idea to avoid the extra complexity. This implies receiving an existing key of the type and also a sorting function compatible with that key, as dates are not sorted as numbers or strings. It does not worth the complexity IMO.